xivo/system.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2008-2022 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2008-2023 The Wazo Authors  (see the AUTHORS file)
```

Comment:
I think most of this file is unused, could you check and delete it if it is unused?